sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -1315,18 +1315,18 @@ public void updateExternalData(List<ExternalDataDto> externalData) throws Extern
 	}
 
 	public void updateCompleteness(String caseUuid) {
-		Case caze = getByUuid(caseUuid);
-		caze.setCompleteness(calculateCompleteness(caze));
-		ensurePersisted(caze);
+		updateCompleteness(getByUuid(caseUuid));
 	}
 
 	@TransactionAttribute(TransactionAttributeType.REQUIRES_NEW)
 	public void updateCompleteness(List<String> caseUuids) {
 		List<Case> casesForUpdate = getByUuids(caseUuids);
```

Comment:
I meant to not load x cases at once, but one by one to release not used entities again.

Reply:
Discussed with @sergiupacurariu: We leave it as-is, JPA cache footprint does not change